### ISSUE TITLE
Cant silent completely the log

### ISSUE THREAD

**Issue by hamidnahli** [SergeyPirogov/webdriver_manager#287]
with `ChromeDriverManager("2.26", log_level=0, print_first_line=False).install()` I still get the log of 
```
====== WebDriver manager ======
Driver [/Users/testuser/.wdm/drivers/chromedriver/mac64/2.26/chromedriver] found in cache
```
How do I silent that log too?

**Comment by mdp18** [comment]
Those logs only show up to me if log_level is not 0.. Make sure you're calling the correct ChromeDriveManager at runtime.

**Comment by dwightmulcahy** [comment]
see #277 as there seems to be more activity there.

**Comment by aleksandr-kotlyar** [comment]
@hamidnahli @mdp18 @dwightmulcahy hello! I have made a feature flag. Please check out at master branch.

https://github.com/SergeyPirogov/webdriver_manager/blob/1281af37ef4fb3fa2e30231487d56cb535216a1b/README.md#L207-L209

This should work and would not affect your project loggers.

**Comment by aleksandr-kotlyar** [comment]
Fixed in 3.5.4 already in pypi.
